lib/miq_automation_engine/service_models/miq_ae_service_container_image.rb

```diff
@@ -13,6 +13,7 @@ class MiqAeServiceContainerImage < MiqAeServiceModelBase
     expose :openscap_rule_results,        :association => true
     expose :exposed_ports,                :association => true
     expose :environment_variables,        :association => true
+    expose :image_ref
```

Comment:
I was able to get the digest in my automate script using `$evm.root['container_image'].digest` (thanks to @gmcculloug). I'm not sure if it's reliable data (i.e. available for EVERY image) but worked in my limited testing.